spec/mongo/uri/srv_protocol_spec.rb

```diff
@@ -713,19 +713,6 @@
             expect(client.options[:auth_source]).to eq(source)
           end
         end
-
-        context '$external' do
```

Comment:
Do you think there is no value in keeping these tests?

Reply:
There are still tests starting on line 702 that test that an auth source is set on the client as passed in on the URI string. Since there's no longer any special treatment for `$external`, this test seems unnecessary.

Reply:
OK sounds good.